### PR TITLE
Testing entry ejs

### PR DESCRIPTION
**Pull request by ryanboothforever**
A better looking Entry EJS.  I will come back and fix the like/delete in a bit.

### CODE REVIEW

**Comment by AngelGDev** [comment]
Looks good. I'd like it to have a link to go back to the feed, a space to display the caption of the entry, and maybe the blue banner on top to be more consistent with the rest of the app

**Comment by laylar** [comment]
Screen Shot 2021-05-04 at 2 47 51 PM
This is what I see on the profile page on the testingEntryEjs branch. Is that what it's supposed to look like? If so, I think this is ready to merge.

**Review by laylar — CHANGES_REQUESTED** [review]
This actually breaks it. It was working with "entries" before because it is looping through entries, now it's trying to loop through "entry".

File: views/feed.ejs [hunk 1]
```diff
@@ -1,16 +1,53 @@
...
+            </div>
+          </div>
+          <a href="/entries/<%= entry[i]._id%>">
+            <img class="feed-img" src="<%= entries[i].image%>">
+          </a>
+          <div>
+            <span class="fa fa-heart"></span>
+            <span><%= entry[i].likeCount %></span>
+          </div>
+        </li>
```
Comment (laylar) [inline views/feed.ejs]:
```suggestion
        <% for(var i=0; i<entries.length; i++) {%>
        <li class="card">
          <div class="entry-topper">
            <div class="entry-title">
              <p><%= entries[i].title%></p>
            </div>
            <div class="entry-author">
              <p>By: <%= entries[i].author%></p>
            </div>
          </div>
          <a href="/entries/<%= entries[i]._id%>">
            <img class="feed-img" src="<%= entries[i].image%>">
          </a>
          <div>
            <span class="fa fa-heart"></span>
            <span><%= entries[i].likeCount %></span>
          </div>
        </li>
```

**Review by laylar — CHANGES_REQUESTED** [review]
I went in and suggested the specific lines where the edits should be. Mostly trying out that functionality, I think you'll know what I mean when you see the previous comment.

**Comment by AngelGDev** [comment]
Inside feed.ejs we use entries[i]._property_ while looping through the array.
Inside entry.ejs we use entry._proprety_

Sorry if I didn't make that clear before

**Pull request merged**